src/main/java/org/trips/service_framework/configs/SnsConfig.java

```diff
@@ -0,0 +1,27 @@
+package org.trips.service_framework.configs;
+
+import com.amazonaws.services.sns.AmazonSNS;
+import com.amazonaws.services.sns.AmazonSNSClientBuilder;
+import lombok.Getter;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+
+import javax.annotation.PostConstruct;
+
+@Getter
+@Configuration
```

Comment:
why is this class annotated with @Configuration if no bean is defined? this is not the right way to write config.